src/components/Header.js

```diff
@@ -38,7 +40,7 @@ const Header = () => (
       <div>
         <a href="/" className="button">Home</a>
         <a href="/post" className="button">New Post +</a>
-        <a href="/login" className="button" onClick={window.sessionStorage.setItem('logout', true)}>Logout</a>
+        <a href="/login" className="button" onClick={() => {Cookies.remove('accountID')}}>Logout</a>
```

Comment:
Let's not render the Logout button unless there is an account ID cookie to remove. You can do that inline with 
```
Cookies.get('accountID') ? (
    <a href="/login" className="button" onClick={() => {Cookies.remove('accountID')}}>Logout</a>
) : null
```